src/inspector_socket_server.cc

```diff
@@ -234,14 +234,16 @@ void PrintDebuggerReadyMessage(
     const std::string& host,
     const std::vector<InspectorSocketServer::ServerSocketPtr>& server_sockets,
     const std::vector<std::string>& ids,
+    const std::string& verb,
```

Comment:
```suggestion
    const char* verb,
```

Please disregard if I'm missing something